models/src/main/protobuf/RhoTypes.proto

```diff
@@ -53,6 +53,10 @@ message Var {
     }
 }
 
+message Bundle {
+    repeated Par body = 1;
```

Comment:
It can just be a single Par

Reply:
I thought so and I had initially just `Par body = 1` but then I checked the source code and it was `Option[Par]` -- is that OK? can the `bundle {}` be empty?